packages/debian-package/deb-helper.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+#  Copyright (C) 2007 National Institute For Space Research (INPE) - Brazil.
+#
+#  This file is part of TerraMA2 - a free and open source computational
+#  platform for analysis, monitoring, and alert of geo-environmental extremes.
+#
+#  TerraMA2 is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation, either version 3 of the License,
+#  or (at your option) any later version.
+#
+#  TerraMA2 is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with TerraMA2. See LICENSE. If not, write to
+#  TerraMA2 Team at <terrama2-team@dpi.inpe.br>.
+#
+#
+#  Description: Install TerraMa2 Helper on Linux Ubuntu.
+#
+#  Author: Carolina Galvão dos Santos
+#
+#
+#  Example:
+#  $ ./deb-helper.sh
+#
+
+export TMVERSION=4.0.0
+export DEBNAME=terrama2-helper
+export DEBFILENAME=terrama2-helper
+export DEBARC=amd64
+export HELPER_FOLDER=`pwd`/../../helper
+export TM_FOLDER=opt/terrama2/${TMVERSION}/webapp
```

Comment:
@carolgsantos please confirm, isn't it missing a "/" at the beginning of the line?

Reply:
@janosimas this is correct, take a look at lines 43 and 85 where it's used.